core/templates/components/question-directives/question-player/question-player.component.html

```diff
@@ -33,8 +37,8 @@ <h4 class="practice-questions-coming-soon-header">Coming Soon!</h4>
   <!-- Results Page -->
   <div class="oppia-question-player-results d-flex flex-column" *ngIf="resultsLoaded">
     <div class="oppia-result-header">
-      <h3 class="oppia-question-result-header e2e-test-practice-complete-message" *ngIf="testIsPassed" translate="I18N_QUESTION_PLAYER_TEST_PASSED"></h3>
-      <h3 class="oppia-question-result-header" *ngIf="!testIsPassed" translate="I18N_QUESTION_PLAYER_TEST_FAILED"></h3>
+      <h3 class="oppia-question-result-header e2e-test-practice-complete-message" *ngIf="testIsPassed" translate="I18N_QUESTION_PLAYER_TEST_PASSED" tabindex="0"></h3>
```

Comment:
```suggestion
      <h3 class="oppia-question-result-header e2e-test-practice-complete-message"
          *ngIf="testIsPassed"
          tabindex="0">
       {{ 'I18N_QUESTION_PLAYER_TEST_PASSED' | translate }}
      </h3>
```
Here and ditto elsehwere